style.css

```diff
@@ -437,10 +419,127 @@ a {
   flex-direction: column;
   align-items: center;
   font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: normal;
   font-size: 10px;
   line-height: 43px;
   color: #fff;
   padding-top: 2rem;
 }
+
+.form-section {
+  height: 800px;
+}
+
+.form-h1 {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 27px;
+  line-height: 35px;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- [ ] The instructions say that you should follow the given design, in this case, the background color may differ depending on the given design. See the difference.

![your-design](https://user-images.githubusercontent.com/53124548/129777180-f136998a-a137-4627-9cd9-18da035945d0.png)
![given](https://user-images.githubusercontent.com/53124548/129777218-de9556a2-929e-44a5-82ec-7f1e02147684.png)

Reply:
Thank you for your feedback @elisha2kyakpo1. We have different templates for this project. The image above is an example of the first template which has that particular background image. I am working on the second template which has a plain background.

Reply:
Hey, @Whoistolu, sorry for that, if that's the case then you can make a screen shot and put it under the PR to avoid other reviewers making the same comment